packages/cli/src/ui/hooks/useGeminiStream.ts

```diff
@@ -450,6 +452,16 @@ export const useGeminiStream = (
     [addItem, config],
   );
 
+  const handleLoopDetectedEvent = useCallback(() => {
+    addItem(
+      {
+        type: 'info',
+        text: `A potential loop was detected, which may be due to repetitive tool calls or other model behavior. The request has been cancelled. Please try again with a more specific prompt.`,
```

Comment:
Updating verbiage a little bit because the existing outputed content will still be in the chat history so canceled felt a bit strong and "try again" did as well.

```suggestion
        text: `A potential loop was detected. This can happen due to repetitive tool calls or other model behavior. The request has been halted.`,
```